Src/Fido2/Extensions/CryptoUtils.cs

```diff
@@ -139,7 +139,7 @@ public static byte[] SigFromEcDsaSig(byte[] ecDsaSig, int keySize)
 
         // Create buffer to copy R into 
         Span<byte> p1363R = coefficientSize <= 64
-            ? stackalloc byte[coefficientSize]
+            ? (stackalloc byte[64]).Slice(0, coefficientSize)
```

Comment:
What's happening here? Why are we always allocating 64 bytes now?

Reply:
It can be slightly faster to allocate small regions from the stack using a const value and slicing it to size than to use a variable. 

There's a bit of discussion on this here: https://github.com/dotnet/docs/issues/28823 and a general recommendation here: https://vcsjones.dev/stackalloc/.

However, it looks like there's some debate on the thresholds where this is an improvement, and whether the JIT can be improved in the future to make this irrelevant.

https://github.com/dotnet/runtime/issues/89478

Given that our preceding <= check makes the safety concern irrelevant, the performance impact is 1-2ns, and that this might be optimized in the future -- will revert this for clarity.